### ISSUE TITLE
BinaryReader in the Python API does not have a ReadCString method

### ISSUE THREAD

**Issue by xusheng6** [Vector35/binaryninja-api#4642]
We have it in the C++ API, but it is implemented in the C++ API. We will need to do the same in the Python API

**Comment by psifertex** [comment]
Isn't that just: 
https://api.binary.ninja/binaryninja.binaryview-module.html#binaryninja.binaryview.BinaryView.get_ascii_string_at
?

**Comment by plafosse** [comment]
No. When you have a BinaryReader object you're usually parsing things out of a RawView to create a view. In C++ api we have a `string BinaryReader::ReadCString(size_t maxSize)` the python API doesn't have the equivalent.  Though looking at the C++ API side implementation its quite poor and we should really expose the one in the core instead. Since we moved our binaryviews to use the API I'm betting this is having some significant performance impacts and so we should prioritize this fixe.
